src/settings/routes/PickListSettings.js

```diff
@@ -17,13 +17,20 @@ export default class PickListSettings extends React.Component {
     this.connectedControlledVocab = props.stripes.connect(ControlledVocab);
   }
 
+  suppressDelete = (category) => {
+    const { internal, values = [] } = category;
+
+    return !(!internal && !values.length);
```

Comment:
[Dust off that discrete math!](https://en.wikipedia.org/wiki/Negation#Double_negation). 
```suggestion
    return internal || values.length;
```